src/Editor/GUI/ResourceSelector.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "../Resources.hpp"
+
+namespace GUI {
+    /// Used to select a resource from the resource list.
+    class ResourceSelector {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The class 'ResourceSelector' does not have a constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AnoConstructor)